tools/ktx/command_encode.cpp

```diff
@@ -72,16 +74,12 @@ Encode a KTX2 file.
     - Daniel Rákos, RasterGrid www.rastergrid.com
 */
 class CommandEncode : public Command {
-    enum {
-        all = -1,
-    };
-
     struct OptionsEncode {
         void init(cxxopts::Options& opts);
         void process(cxxopts::Options& opts, cxxopts::ParseResult& args, Reporter& report);
     };
 
-    Combine<OptionsEncode, OptionsCodec<true>, OptionsMetrics, OptionsCompress, OptionsSingleInSingleOut, OptionsGeneric> options;
+    Combine<OptionsEncode, OptionsBasis<true>, OptionsCodecCommon, OptionsMetrics, OptionsCompress, OptionsSingleInSingleOut, OptionsGeneric> options;
```

Comment:
OptionsCompress and everything related needs to be renamed OptionsDeflate. You've only changed the file name.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/817/commits/dd12a5ec82f53758c77cbf18c7c2f6e2443a30ff